### PR TITLE
[RFC] Adds .anim-drop Animation Utility Class

### PR DESCRIPTION
**Pull request by keithpops**
In use for the ```ActionBar```:

anim-move

### CODE REVIEW

**Comment by tsargent** [comment]
hot :+1:

File: src/scss/animations/_move.scss [hunk 1]
```diff
@@ -0,0 +1,12 @@
+.anim-move {
+  margin-top: -$space-3;
+  opacity: 0;
+  animation-name: move;
+  animation-duration: 0.2s;
+  animation-fill-mode: forwards;
+}
+
+@keyframes move {
+    0% { margin-top: -20px; opacity: 0; }
```
Comment (HipsterBrown) [inline src/scss/animations/_move.scss]: `margin` is not a GPU supported animation, which can lead to "jank". http://www.html5rocks.com/en/tutorials/speed/high-performance-animations/

See also: http://csstriggers.com/#margin v. http://csstriggers.com/#transform

The same animation should work with `transform: translateY(-100%)` to `transform: translateY(0%)` and keep it buttery smooth on all devices.

File: src/scss/animations/_move.scss [hunk 2]
```diff
@@ -0,0 +1,12 @@
+.anim-move {
+  margin-top: -$space-3;
+  opacity: 0;
+  animation-name: move;
+  animation-duration: 0.2s;
+  animation-fill-mode: forwards;
+}
```
Comment (HipsterBrown) [inline src/scss/animations/_move.scss]: Properties should be ordered alphabetically, according to the `scss` coding styleguide.

File: src/scss/animations/_move.scss (hunk 1 above)
Comment (keithpops) [inline src/scss/animations/_move.scss]: That's also why we use it on the directional buttons. I'll adjust

**Comment by HipsterBrown** [comment]
Is `anim-move` the best description of this utility? "Move" is so general. Would something like `anim-drop` or `anim-drop-in` be the more descriptive of what's actually happening?

File: src/scss/animations/_move.scss (hunk 1 above)
Comment (HipsterBrown) [inline src/scss/animations/_move.scss]: The `20px` measurement looks very specific to the View Component as well. Using the relative units like percentages will keep this utility general.

**Comment by keithpops** [comment]
@HipsterBrown I'm no unopposed to it but this is something that should be addressed with other animations too if we are giving optional directions.

File: src/scss/animations/_move.scss (hunk 1 above)
Comment (keithpops) [inline src/scss/animations/_move.scss]: That was an oversight where -$space-3 was supposed to go. I understand that.

**Comment by HipsterBrown** [comment]
Agreed. We have a convention based on Brent Jackson's VHS lib, with an animation utility going one way and the opposite direction has the utility name appended with `-reverse`. Do we want to append directions for certain animations as well? I.e. `anim-fade-left`

**Comment by HipsterBrown** [comment]
:+1:

**Pull request merged**